spec/models/manageiq/providers/embedded_terraform/automation_manager/configuration_script_source_spec.rb

```diff
@@ -1,3 +1,5 @@
+TERRAFORM_RUNNER_URL = 'https://1.2.3.4:7000'.freeze
```

Comment:
We shouldn't add constants to the global scope, a better way to do this is to have a `let(:terraform_runner_url) { 'https://1.2.3.4:7000' }` inside your `RSpec.describe` block so it is available to every test in this file but not globally.

Reply:
E.g. you can see it is causing issues just while running the tests
```
** ManageIQ master, codename: Spassky
/home/grare/adam/src/manageiq/manageiq-providers-embedded_terraform/spec/models/manageiq/providers/embedded_terraform/automation_manager/configuration_script_source_spec.rb:1: warning: already initialized constant TERRAFORM_RUNNER_URL
/home/grare/adam/src/manageiq/manageiq-providers-embedded_terraform/spec/lib/terraform/runner_spec.rb:4: warning: previous definition of TERRAFORM_RUNNER_URL was here
```

Reply:
Fixed